VISION.md

```diff
@@ -1,12 +1,12 @@
 # OpenEBS Umbrella Project Vision
 
 > [!Important]
-> OpenEBS is an "umbrella" project,  composed as a federation of individual sub projects (repositories). The umbrella project, every sub project, repository/repo and file in the OpenEBS organization adopts and follows the same set of umbrella policies in the OpenEBS Community repo. This is the Umbrella Vision for OpenEBS.
+> OpenEBS is an "umbrella" project,  composed as a federation of individual sub-projects (repositories). The umbrella project, every sub-project, repository/repo, and file in the OpenEBS organization adopts and follows the same set of umbrella policies in the OpenEBS Community repo. This is the Umbrella Vision for OpenEBS.
 <BR>
 
 OpenEBS (Open-Elastic-Block-Storage) aims to provide fast, resilient, available, enterprise-grade storage; natively inside a Kubernetes cluster as Hyper-Converged storage. OpenEBS Users provision OpenEBS storage services to Kubernetes applications running natively inside a Kubernetes cluster.
 
-In this document we present the project:
+In this document, we present the project:
```

Comment:
The whole phrase should be changed from passive to active tense "This document presents the project::

Reply:
Done.